SI/Sem.04/Pract.04/Task02.cpp

```diff
@@ -0,0 +1,25 @@
+using namespace std;
+#include <iostream>
+
+bool isEven(int num)
+{
+    return num % 2 == 0 ? true : false;
```

Comment:
num % 2 == 0 is a boolean expression, so it returns a boolean value that you can directly return